src/routes/(app)/admin[[hash=admin_hash]]/file/+page.server.ts

```diff
@@ -0,0 +1,7 @@
+import { collections } from '$lib/server/database';
+
+export const load = async () => {
+	return {
+		pictures: await collections.digitalFiles.find({}).toArray()
```

Comment:
```suggestion
		pictures: await collections.digitalFiles.find({productId: {$exists: false}}).toArray()
```

Similar to pictures, only display the digital files with no product associated.

`productId` should also be made optional on a digital file